### PR TITLE
General Remarks

### PR DESCRIPTION
**Pull request by matteopilz**
#### Brief description of what is fixed or changed

I'm lacking permissions here, but these changes are connected to https://github.com/OpenMS/openms.de/issues/60
and https://github.com/OpenMS/openms.de/issues/61

- [x] Updated text in about section
- [x] Added issue filter for community page
- [x] Added twitter linkt to community page
- [x] Updated workflow image on main page (config.yaml) - (updated theme: https://github.com/OpenMS/scientific-python-hugo-theme/pull/1)
- [ ] (Split workflow image into three separate ones)
- [x] Replaced link to Galaxy in "Get Started" on main page with link to workflow overview
- [x] Replaced "Get help" with "Help"
- [x] Restructured installation page
- [x] Changed layout of h2 and h3 headers

### CODE REVIEW

File: content/en/config.yaml [hunk 1]
```diff
@@ -30,8 +30,8 @@ params:
       subtitle: Modify and create your own OpenMS workflows in KNIME.
       image: knime-edit.png
       buttonText: Get Started
-      buttonLink: "https://openms.readthedocs.io/en/latest/run-workflows-with-openms-tools/overview.html"
-    - title: Execute python bindings 🐍
+      buttonLink: "https://openms.readthedocs.io/en/latest/run-workflows-with-openms-tools/openms-in-knime.html"
+    - title: Execute python bindings
```
Comment (timosachsenberg) [inline content/en/config.yaml:34]: bindings are not executed
```suggestion
    - title: Use OpenMS in python
```

**Comment by matteopilz** [comment]
@tjeerdijk, can you take a look, so we can wrap this up? I think you had some concerns with using the image of all workflow logos. @rahul799, did you take a look if we can put h1 level headers in the sidebar as well?

**Comment by timosachsenberg** [comment]
@jpfeuffer do you know if we can use these logos (e.g., mainly KNIME seems to be picky)?

**Comment by matteopilz** [comment]
Not a lawyer, but I would say that we can.
https://de.wikipedia.org/wiki/Datei:KNIMELogoTM.svg
https://galaxyproject.org/images/galaxy-logos/
https://github.com/nextflow-io/trademark
https://nf-co.re/developers/design_guidelines

**Comment by jpfeuffer** [comment]
Can't we insert separate logos, so we could actually reuse them in other places?
Also updating would be much easier!

**Comment by poshul** [comment]
@tjeerdijk would you have a chance to take a look at this today?

**Comment by matteopilz** [comment]
@jpfeuffer, can you check if everything is correct?

**Comment by timosachsenberg** [comment]
@jpfeuffer can you give it a quick look? screenshots wotj errors are already creeping into the paper :P

File: content/en/about.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ sidebar: false
 
 _Some information about the OpenMS project and community_
 
-OpenMS is an open-source software C++ library for LC-MS data management and analyses. It offers an infrastructure for rapid development of mass spectrometry related software. OpenMS is free software available under the three clause BSD license and runs under Windows, macOS, and Linux. OpenMS will always be 100% open source software, free for all to use and released under the liberal terms of the [modified BSD license](https://github.com/openms/openms/blob/main/License.txt).
+OpenMS is a free open-source software C++ library for LC-MS data management and analyses. It offers an infrastructure for rapid development of mass spectrometry related software. OpenMS is available under a modified three clause [BSD license](https://github.com/openms/openms/blob/main/License.txt) and runs under Windows, macOS, and Linux.
```
Comment (timosachsenberg) [inline content/en/about.md:8]:
```suggestion
OpenMS is an open-source software C++ library for LC-MS data management and analyses. It offers an infrastructure for rapid development of mass spectrometry related software. OpenMS is available under a modified three clause [BSD license](https://github.com/openms/openms/blob/main/License.txt) and runs under Windows, macOS, and Linux.
```

File: content/en/communication.md [hunk 3]
```diff
@@ -3,67 +3,85 @@ title: Community
...
 
-A real-time chat room to ask questions about _contributing_ to OpenMS.
-This is a private space, specifically meant for people who are hesitant to
-bring up their questions or ideas on a large public mailing list or GitHub.
+Discord allows the users to communicate in different channels, publicly as well as privately
 
 ***
 
-### <a class="button cta rounded primary-btn raised" href="https://discord.gg/aJyWqf6uCn">Discord</a>
+### <a class="button cta rounded primary-btn raised" href="https://github.com/openms">GitHub</a>
```
Comment (timosachsenberg) [inline content/en/communication.md:30]:
```suggestion
### <a class="button cta rounded primary-btn raised" href="https://github.com/OpenMS">GitHub</a>
```

Comment (timosachsenberg) [reply content/en/communication.md:30]: works also in lower case as it seems

File: content/en/config.yaml [hunk 4]
```diff
@@ -20,41 +20,41 @@ params:
   - title: "OpenMS allows users to:"
     bgColor: ECF2F9
     heroItems:
-    - title: Run existing workflows
+    - title: Run existing workflows with KNIME
```
Comment (timosachsenberg) [inline content/en/config.yaml:23]:
```suggestion
    - title: Run existing workflows
```

File: content/en/communication.md [hunk 5]
```diff
@@ -3,67 +3,85 @@ title: Community
...
+This is a private space, specifically meant for people who are hesitant to
+bring up their questions or ideas on a large public mailing list or GitHub.
 
 ***
 
 ### <a class="button cta rounded primary-btn raised" href="https://sourceforge.net/p/open-ms/mailman/open-ms-announcements">OpenMS mailing list</a>
 
 
-This list is the main forum for longer-form discussions, like adding new features to OpenMS, making changes to the OpenMS Roadmap, and all kinds of project-wide decision making.
+These lists are the main form of receiving OpenMS related updates, like new features, changes to the roadmap, and all kinds of project-wide decision making.
```
Comment (jpfeuffer) [inline content/en/communication.md:69]:
```suggestion
These lists are the main form of receiving OpenMS-related updates, like new features, changes to the roadmap, and all kinds of project-wide decision making.
```

File: themes/scientific-python-hugo-theme
Comment (timosachsenberg) [inline themes/scientific-python-hugo-theme:1]: what about these files?

File: content/en/tutorials.md [hunk 6]
```diff
@@ -26,19 +26,24 @@ This tutorial was designed to guide you through the use of our Community Contrib
...
 {{< /button >}}
 
 For use with the binary installers of OpenMS. It describes how to use the OpenMS TOPP tools from the command line and has an extensive tutorial on visualisation with TOPPView. You need your own data to work through it. In case you do not have any, have a look at the data from the User Tutorial above.
 
-### Old versions
-Old versions are available in the documentation of older OpenMS releases [here](http://ftp.mi.fu-berlin.de/pub/OpenMS/).
+## Additional Resources
+
+### Tutorials Archive
+We will release new versions on our [GitHub page for Tutorials](https://github.com/OpenMS/Tutorials/releases). Older versions can be found there. It also contains releases for the versions used in different hands-on workshops (if you want to replicate it at home). The repository does not include data. Unless otherwise noted, the tutorials use the same (or a subset) of the tutorial data above.
```
Comment (jpfeuffer) [inline content/en/tutorials.md:42]: Not sure if I would link to releases. If so, we really need to make sure to create releases, not just tag the repo.

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

File: content/en/config.yaml [hunk 7]
```diff
@@ -20,41 +20,41 @@ params:
...
       subtitle: Run existing workflows for proteomics and metabolomics with OpenMS in KNIME.
       leftAlign: true
       image: knime-analytics.png
       buttonText: Get Started
-      buttonLink: "https://hub.knime.com/search?type=Workflow&tag=OpenMS&sort=maxKudos"
-    - title: Create workflows
-      subtitle: Modify and create your own OpenMS workflows in KNIME.
+      buttonLink: "https://openms.readthedocs.io/en/latest/run-workflows-with-openms-tools/openms-in-knime.html"
```
Comment (jpfeuffer) [inline content/en/config.yaml:28]: This does not exist anymore.

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

File: content/en/tutorials.md (hunk 6 above)
Comment (timosachsenberg) [reply content/en/tutorials.md:42]: k let's keep this low maintenance and let's not link to releases

File: content/en/config.yaml (hunk 4 above)
Comment (matteopilz) [reply content/en/config.yaml:23]: The idea here was to have some kind of hierarchy in difficulty. KNIME workflows are / should be the most basic use case of OpenMS.

Comment (timosachsenberg) [reply content/en/config.yaml:23]: not if we have web apps :P

Comment (timosachsenberg) [reply content/en/config.yaml:23]: I just find it a bit redundant to have it in the title. Another reason: the acronyme KNIME doesn't add information to beginners so we can leave it out in the title (but still have it in the subtitle).

Comment (matteopilz) [reply content/en/config.yaml:23]: We need to restructure the website anyways, there is still a lot out of order or unintuitive. Question is, should we do this later or merge these changes first. The link and reference to workflows in general is in create your own workflows, one step later.

Comment (matteopilz) [reply content/en/config.yaml:23]: We wanted to have some distinction between this step and the next.

Comment (timosachsenberg) [reply content/en/config.yaml]: I just wanted to chang the title not the subtitle. It still refers to knime so we still have the distinction.

Comment (timosachsenberg) [reply content/en/config.yaml]: I just wanted to chang the title not the subtitle. It still refers to knime so we still have the distinction.

Comment (jpfeuffer) [reply content/en/config.yaml]: The problem is, the way our docs are currently structured the workflow system is on the top-level.
So there will not be an "existing workflows" overview page for all workflow systems. The alternative
is to restructure the docs by topic and on every page have a distinction between the workflow systems.
Also not great at all. So I think just silently picking KNIME here is fine.

But: I am not sure if a super new user knows what to do on the KNIME hub if KNIME was not even introduced yet.

So maybe the restructure is really necessary.

Just saying: We are discussing this for the fifth time or so. How about making a final decision in a larger meeting? There is always someone coming up who doesn't like a certain part but it is probably not possible to satisfy every concern.

File: content/en/config.yaml [hunk 8]
```diff
@@ -132,7 +142,7 @@ footer:
         - text: Install
           link: /install
```
[the remark this answers is not in the document]
Comment (matteopilz) [reply content/en/config.yaml:143]: Just noticed, that we have one navigation bar install link and two extra on the main page.

File: content/en/config.yaml (hunk 4 above)
Comment (timosachsenberg) [reply content/en/config.yaml]: good point - this is turning into turning into bike shedding 
I commited and resolved this issue.

File: content/en/tutorials.md [hunk 9]
```diff
@@ -26,19 +26,24 @@ This tutorial was designed to guide you through the use of our Community Contrib
...
+
+### Documentations Archive
+https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/release/
+
+### Proteomics & Metabolomics Background Material
```
[the remark this answers is not in the document]
Comment (matteopilz) [reply content/en/tutorials.md:47]: Maybe it's best then to remove the tutorials page or link to the documentation again (better just remove it though).

File: themes/scientific-python-hugo-theme
Comment (matteopilz) [reply themes/scientific-python-hugo-theme:1]: Those are just changed formatting, which I think looks slightly better.

File: content/en/tutorials.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (timosachsenberg) [reply content/en/tutorials.md:38]: k can you double check that information is available in the documentation and then remove it? @jpfeuffer any concerns removing it?

**Comment by matteopilz** [comment]
Ok, removed it.

**Comment by timosachsenberg** [comment]
just checked the deployment link... seems install link on the top links to nowhere https://openms.readthedocs.io/en/latest/openms-applications-and-tools/installation.html

**Comment by matteopilz** [comment]
> just checked the deployment link... seems install link on the top links to nowhere https://openms.readthedocs.io/en/latest/openms-applications-and-tools/installation.html

Not yet, the changes are connected to Julianus' restructure https://github.com/OpenMS/OpenMS-docs/pull/198

**Comment by timosachsenberg** [comment]
link now working?

**Review by timosachsenberg — APPROVED** [review]

**Comment by timosachsenberg** [comment]
@jpfeuffer merge?

**Review by jpfeuffer — APPROVED** [review]

**Comment by timosachsenberg** [comment]
Nice!

**Pull request merged**